### ISSUE TITLE
Python score() cannot take bos/eos arguments

### ISSUE THREAD

**Issue by jiaeyan** [kpu/kenlm#232]
I have successfully installed and compiled KenLM on Mac, and trained an arpa file.

However, when I tried to query probability of a string with score() in Python and passed 'bos=False' and 'eos=False' to the function, the error occurred:
"TypeError: score() takes no keyword arguments"

Any idea how to fix this? Thanks!

**Comment by kpu** [comment]
I am unable to reproduce this issue, admittedly on Linux.
```
pip install --user https://github.com/kpu/kenlm/archive/master.zip
python3
Python 3.5.2 (default, Nov 12 2018, 13:43:14) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import kenlm
>>> l = kenlm.Model("../lm/test.arpa")
Loading the LM will be faster if you build a binary file.
Reading /home/heafield/kenlm/lm/test.arpa
----5---10---15---20---25---30---35---40---45---50---55---60---65---70---75---80---85---90---95--100
****************************************************************************************************
>>> l.score("hello")
-23.440101623535156
>>> l.score("hello", bos=False)
-23.025127410888672
>>> l.score("hello", bos=False, eos=True)
-23.025127410888672
```

**Comment by kpu** [comment]
Does `example.py` run correctly?

**Comment by jiaeyan** [comment]
Hi, Ken, thank you very much for replying, and the issue is resolved!

I cloned your repo and installed the setup.py from the python folder, instead of installing it by "pip install --user https://github.com/kpu/kenlm/archive/master.zip". 

Not sure if this is the reason, since I noticed your recent commit shows the cpp "ScoreSentence" function doesn't take eos/bos. Anyway, when I installed by your way, it worked, thanks!
